### PR TITLE
Ab/add closing disclosure

### PR DESCRIPTION
**Pull request by angbonilla**
- configuration for pgs 1-3 for both refinance and purchase loan purposes
- sections containing configured fields include but are not limited to loan costs, other costs, calculating cash to close, summaries of transactions, and payoffs & payments
- both refinance and purchase samples have been added

### CODE REVIEW

**Comment by ethulia** [comment]
The purchase sample PDF is corrupted -- can you check that again?

**Comment by angbonilla** [comment]
@ethulia I wasn't able to utilize any of the table methods for pg 2, but I was able to add tables to pg 3

File: README.md [hunk 1]
```diff
@@ -141,6 +141,9 @@ Find the ACORD year/month at the bottom left corner of each ACORD page. Often th
 - [Gusto](./paystubs/gusto/gusto.json)
 - [Paylocity](./paystubs/paylocity/paylocity.json)
 
+### Closing Disclosures
+- [IDS](./mortgage/closing_disclosure/closing_disclosure.json)
```
Comment (ethulia) [inline README.md:145]: Still says IDS here

Comment (ethulia) [inline README.md:145]: maybe call this real estate instead of mortgage?

Comment (ethulia) [reply README.md:145]: Also this is still named IDS here

**Review by ethulia — APPROVED** [review]

**Pull request merged**